joomla/joomla-cms
# Restore the ability to re initialise the calendar programmatically

Pull Request for Issue #15251 

### Summary of Changes
Minor changes to the calendar script that allow reinitialising the calendar


### Testing Instructions
Apply this patch
Go to Administrator and edit an existing article.
Open the browser inspector and inspect the **button** of the Start Publishing field.
Then change data-show-others="1"  to "0"
Then run this in the browser's console:
```js
JoomlaCalendar.init(document.querySelector('.field-calendar'));
```
the cal should reinitialise, to confirm open it again you should see the table without the days for prev/next month

Alternative test method (only js, also a hint how to do this client side):
Apply this patch
Go to Administrator and edit an existing article.
Open the browser console and copy/paste the following commands
```js
document.getElementById('jform_publish_up_btn').setAttribute('data-show-others', 0);
JoomlaCalendar.init(document.querySelector('.field-calendar'));
```
the cal should reinitialise, to confirm open it again you should see the table without the days for prev/next month


### Expected result


### Actual result



### Documentation Changes Required
